scss/tokens/_raw.scss

```diff
@@ -267,6 +267,7 @@ $ouds-elevation-y-500: 12px !default;
 // scss-docs-start ouds-raw-font
 $ouds-font-family-brand-default: "Helvetica Neue" !default;
 // $ouds-font-family-brand-tv: "Helvetica Neue LT" !default;
+// $ouds-font-family-monospace-consolas: "Consolas" !default;
```

Comment:
This one can be removed. These tokens are handled on the parser's side which needs to clean up some things too.